scripts/qa_crowbarsetup.sh

```diff
@@ -1964,14 +1964,8 @@ Host node$i
 EOF
     done
 
-    # check for error 500 in app/models/node_object.rb:635:in `sort_ifs'#012
-    curl -m 9 -s $crowbar_api_digest $crowbar_api | \
-        tee /root/crowbartest.out
-    if grep -q "Exception caught" /root/crowbartest.out; then
+    curl -m 9 -s $crowbar_api_digest $crowbar_api/nodes.json || \
```

Comment:
even better just call
`onadmin_is_crowbar_api_available`
and change it to use the .json route